worker-document-interface/src/main/java/com/hpe/caf/worker/document/model/Task.java

```diff
@@ -64,4 +65,17 @@
      * @see Application#getService(Class) Application.getService()
      */
     <S> S getService(Class<S> service);
+
+    /**
+     * Allows to set additional response options for this task.
+     * @param queueName Queue name to post response message to.
+     * @param customData Custom data to set on a response message.
+     */
+    void setResponseOptions(String queueName, Map<String, String> customData);
+
+    /**
+     * Retrieves additional response options for this task.
+     * @return Response options if there are any or {@code null}.
```

Comment:
Why would this ever be `null`?  I assume you were thinking that this should only be available if `setResponseOptions()` has been called.  It would be better to just make it always available and have the setter options available on it.